src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfig.java

```diff
@@ -128,4 +138,32 @@ public ListBoxModel doFillRegionItems() {
         }
         return regions;
     }
+
+    public FormValidation doCheckContainer(@QueryParameter String container){
+        FormValidation ret = FormValidation.ok();
+        if (StringUtils.isBlank(container)){
+            ret = FormValidation.error("The container name cannot be empty");
```

Comment:
More polite to make this a warning so users do not get a red error message thrown in their faces when they are first going to `/configure`.